config/locales/forms/cease_or_revoke_forms/en.yml

```diff
@@ -0,0 +1,35 @@
+en:
+  waste_carriers_engine:
+    cease_or_revoke_forms:
+      new:
+        heading: "Revoke or cease registration %{reg_identifier}"
+        tier:
+          upper: "Upper tier"
+          lower: "Lower tier"
+        registration_type:
+          carrier_dealer: "Carrier and dealer"
+          broker_dealer: "Broker and dealer"
+          carrier_broker_dealer: "Carrier, broker and dealer"
+        reason:
+          label: "Reason"
+          help_text: "Maximum 500 characters"
+        cease_or_revoke:
+          legend: "What do you want to do?"
+          options:
+            revoke: "Revoke"
+            cease: "De-register"
+          reason: "Reason"
+        total_cost_info: "The total cost will be shown on the next page"
```

Comment:
Not 100% sure we are using this in this context 😁 !

Reply:
😮 nope! We are not!! XD